tests/ResponseDocumentingTest.php

```diff
@@ -96,3 +98,57 @@ public function index()
         return response()->json(['foo' => 'bar']);
     }
 }
+
+test('laravel responder success support', function () {
+    \Illuminate\Support\Facades\Route::get('api/test', [Foo_TestFive::class, 'index']);
+
+    \Dedoc\Scramble\Scramble::routes(fn (Route $r) => $r->uri === 'api/test');
+    $openApiDocument = app()->make(\Dedoc\Scramble\Generator::class)();
+
+    assertMatchesSnapshot($openApiDocument);
+});
+class Foo_TestFive
```

Comment:
Have you tried using anonymous classes instead of declaring new classes throughout the test file? Or it doesn't matter @sojeda?

Reply:
It's the libraries standards